### ISSUE TITLE
Getting "undefined method `to_ruby'" Error on "watchtower start"

### ISSUE THREAD

**Issue by Artforge** [kalbasit/watch_tower#7]
Full Trace.

/Users/Adam/.rvm/gems/ruby-1.9.2-p180/gems/watch_tower-0.0.1.beta10/lib/watch_tower/config.rb:22:in `[]': undefined method`to_ruby' for #Syck::Map:0x0000010106cfc8 (NoMethodError)
    from /Users/Adam/.rvm/gems/ruby-1.9.2-p180/gems/watch_tower-0.0.1.beta10/lib/watch_tower/cli/start.rb:52:in `start'
    from /Users/Adam/.rvm/gems/ruby-1.9.2-p180/gems/thor-0.14.6/lib/thor/task.rb:22:in`run'
    from /Users/Adam/.rvm/gems/ruby-1.9.2-p180/gems/thor-0.14.6/lib/thor/invocation.rb:118:in `invoke_task'
    from /Users/Adam/.rvm/gems/ruby-1.9.2-p180/gems/thor-0.14.6/lib/thor.rb:263:in`dispatch'
    from /Users/Adam/.rvm/gems/ruby-1.9.2-p180/gems/thor-0.14.6/lib/thor/base.rb:389:in `start'
    from /Users/Adam/.rvm/gems/ruby-1.9.2-p180/gems/watch_tower-0.0.1.beta10/bin/watchtower:23:in`<top (required)>'
    from /Users/Adam/.rvm/gems/ruby-1.9.2-p180/bin/watchtower:19:in `load'
    from /Users/Adam/.rvm/gems/ruby-1.9.2-p180/bin/watchtower:19:in`<main>'

**Comment by niko** [comment]
same here

**Comment by kalbasit** [comment]
Do you have the file ~/.watch_tower/config.yml?

Could you try this in irb, pry or in it's own file and paste the output?

``` ruby
require 'rubygems'
require 'active_support/hash_with_indifferent_access'

config = File.expand_path(File.join(ENV['HOME'], '.watch_tower', 'config.yml'))
yaml = YAML.parse_file(config)
puts yaml.to_ruby
```

Thanks

**Comment by kalbasit** [comment]
Could you guys please try the **feature/fix_issue_7** branch?

``` bash
$ git clone git://github.com/TechnoGate/watch_tower.git
$ cd watch_tower
$ git branch feature/fix_issue_7 origin/feature/fix_issue_7
$ git checkout feature/fix_issue_7
$ bundle install
$ WATCH_TOWER_ENV='development' ./bin/watchtower start -f -d
```

I should probably add a section about development on the README :)

Thanks guys

**Comment by kalbasit** [comment]
I've released version 0.0.1.beta11 that should fix this issue.

Please reopen if this issue persists and gist your config file, I couldn't reproduce the issue and I assume the work I've done should prevent any unnoticed errors parsing the config file.
